manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -593,6 +593,62 @@ status:
   conditions: []
   storedVersions: []
 ---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.5.0
+  labels:
+    gatekeeper.sh/system: "yes"
+  name: providers.externaldata.gatekeeper.sh
+spec:
+  group: externaldata.gatekeeper.sh
```

Comment:
Don't we also need to add this to manager-role clusterrole?

Reply:
good catch, added